package.json

```diff
@@ -22,10 +23,14 @@
     "node": ">=0.12"
   },
   "devDependencies": {
+    "@google-cloud/datastore": "^1.0.2",
```

Comment:
What was the rationale for making them only show up in subdirectories? Going forward, I would like this dependency to be kept up to date so that we are integrating with the latest datastore library. Making it a fixtures makes it easy to miss the update.

In fact, I think we should change the semver range here to "*".